runtime-sdk/src/context.rs

```diff
@@ -126,6 +133,12 @@ pub trait Context {
         }
     }
 
+    /// Returns the maximum number of iterations when doing binary search for estimate gas.
+    fn estimate_gas_estimate_gas_search_max_iters(&self) -> u64 {
```

Comment:
Should this be just `estimate_gas_search_max_iters`? Actually should we move this to the `core` module's `LocalConfig` (and add a method to `core::API` for other modules to use)?

Reply:
Could move it to core yes. I actually had this there initially, don't completely recall why I moved it. Probably due to needing it in the EVM module. But adding it to `core::API` does solve that.

Reply:
Fixed!